toml.md

```diff
@@ -601,6 +611,13 @@ Seconds may be omitted, in which case `:00` will be assumed.
 ldt3 = 1979-05-27T07:32
 ```
 
+Valid years run from `0001` to `9999`.
+
+```toml
+# ldt_year0 = 0000-12-31 23:59:59  # INVALID
+ldt_year1 = 0001-01-01 00:00:00    # VALID
+```
+
```

Comment:
I think it's enough to give a single example of valid and invalid years (above), no need to repeat that here or in the following sections.

Reply:
I'd prefer to keep the examples, but see my comments on #1000 first.

Reply:
I did remove this example; the only example now appears in the Offset Date-Time section.